src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,17 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment }) => (
+  <>
```

Comment:
No need to wrap the code into the fragment since it has the common parent element